Buildings/Experimental/DHC/BaseClasses/Steam/PartialSaturatedControlVolume.mo

```diff
@@ -0,0 +1,294 @@
+﻿within Buildings.Experimental.DHC.BaseClasses.Steam;
+partial model PartialSaturatedControlVolume
+  "Partial control volume for evaporation/condensation processes"
+  extends Buildings.BaseClasses.BaseIcon;
+  extends Buildings.Experimental.DHC.BaseClasses.Steam.PartialTwoPortTwoMedium(
+    p_start=1e6,
+    final T_start=MediumSte.saturationTemperature(p_start));
+
+  // Medium declarations
+  replaceable package MediumWat =
+    Buildings.Media.Specialized.Water.TemperatureDependentDensity
+    "Liquid water medium";
+  replaceable package MediumSte = Buildings.Media.Steam
+     "Steam medium";
+
+  // Parameters
+  parameter Modelica.Units.SI.Volume V "Total volume";
+  // Initialization
+  parameter Modelica.Units.SI.Volume VWat_start=V/2
+    "Start value of liquid volume"
+    annotation (Dialog(tab="Initialization"));
+
+  // Variables
+  MediumWat.ThermodynamicState stateWat(p=p, T=T)
+    "Saturated state, liquid water";
+  MediumWat.ThermodynamicState stateSte(p=p, T=T)
```

Comment:
@khinkelman Can you confirm if this should actually be ``MediumSte`` instead of ``MediumWat``?  Making the change helps resolve one of the OpenModelica compilation errors.